pkg/util/fedinformer/genericmanager/multi_cluster_manager.go

```diff
@@ -1,3 +1,5 @@
+//go:generate mockgen -source=multi_cluster_manager.go  -destination=testing/mock_multi_cluster_manager.go -package=testing MultiClusterInformerManager
```

Comment:
Just a question, Do I need to install the `mockgen` command in advance?

Reply:
Yes, this PR too big, check this one first: https://github.com/karmada-io/karmada/pull/2654